src/scenes/Dashboard/EngagementsUsingAIWidget/EngagementsUsingAIGrid.tsx

```diff
@@ -0,0 +1,248 @@
+import {
+  DataGridPro,
+  DataGridProProps as DataGridProps,
+  GridColDef,
+  GridToolbarColumnsButton,
+  GridToolbarFilterButton,
+} from '@mui/x-data-grid-pro';
+import { merge } from 'lodash';
+import { useMemo } from 'react';
+import { SetOptional } from 'type-fest';
+import {
+  AiAssistedTranslationLabels,
+  AiAssistedTranslationList,
+  EngagementFilters,
+  EngagementListInput,
+} from '~/api/schema.graphql';
+import { CalendarDate, extendSx } from '~/common';
+import {
+  booleanColumn,
+  DefaultDataGridStyles,
+  enumColumn,
+  getInitialVisibility,
+  noHeaderFilterButtons,
+  QuickFilterButton,
+  QuickFilterResetButton,
+  QuickFilters,
+  textColumn,
+  Toolbar,
+  useDataGridSource,
+  useFilterToggle,
+} from '~/components/Grid';
+import { LinkColumn } from '~/components/Grid/Columns/LinkColumn';
+import { ProjectNameColumn } from '~/components/Grid/Columns/ProjectNameColumn';
+import { Link } from '~/components/Routing';
+import {
+  EngagementUsingAiDataGridRowFragment as EngagementUsingAI,
+  EngagementUsingAiListDocument,
+} from './engagementsUsingAIDataGridRow.graphql';
+
+export type EngagementUsingAIColumnMapShape = Record<
+  string,
+  SetOptional<GridColDef<EngagementUsingAI>, 'field'>
+>;
+
+export const EngagementUsingAIColumnMap = {
+  project: ProjectNameColumn({
+    field: 'project.name',
+    valueGetter: (_, engagement) => engagement.project,
+  }),
+  language: {
+    headerName: 'Language / Intern',
+    field: 'nameProjectLast',
+    ...textColumn(),
+    width: 200,
+    valueGetter: (_, row) => {
+      return row.__typename === 'LanguageEngagement'
+        ? row.language.value?.name.value
+        : row.__typename === 'InternshipEngagement'
+        ? row.intern.value?.fullName
+        : null;
+    },
+    renderCell: ({ value, row }) => {
+      return row.__typename === 'LanguageEngagement' ? (
+        <Link to={`/languages/${row.language.value!.id}`}>{value}</Link>
+      ) : row.__typename === 'InternshipEngagement' ? (
+        <Link to={`/users/${row.intern.value!.id}`}>{value}</Link>
+      ) : null;
+    },
+    hideable: false,
+    serverFilter: (value): EngagementFilters => ({ engagedName: value }),
+  },
+  viewEngagement: LinkColumn({
+    field: 'Engagement',
+    headerName: '',
+    valueGetter: (_, engagement) => engagement,
+    destination: (id) => `/engagements/${id}`,
+  }),
```

Comment:
Similar to my comment on the GQL fragment all of this is copy/pasted from the other engagement grid. It would be nice if referenced them.
Maybe that's via a pick?
```ts
export const EngagementUsingAIColumnMap = {
  ...pick(EngagementColumnMap, ['project', 'language', 'id'])
  ...
  ...pick(EngagementColumnMap, ['isMember', 'pinned'])
};
```